app/forms/idv/phone_form.rb

```diff
@@ -31,7 +31,8 @@ def submit(params)
     attr_writer :phone
 
     def update_idv_params(phone)
-      idv_params[:phone] = phone
+      normalized_phone = phone.gsub(/([-()+ ])/, '')[1..-1]
```

Comment:
Yes, just removing non-digit characters. Good call about the `\D` matcher. TIL. I don't think we need an entire gem for this normalization. Because we are formatting the phone number in the form object using the `phony_rails` gem, we know for sure that any number we will be normalizing will always be in this format: `+1 (area code) XXX-YYYY`. So, all we need to do is strip all non-numeric characters, then drop the country code.